digitaltwin_client/inc/digitaltwin_client_common.h

```diff
@@ -50,7 +50,7 @@ typedef void* DIGITALTWIN_INTERFACE_CLIENT_HANDLE;
         DIGITALTWIN_CLIENT_ERROR_HANDLE_DESTROYED,              \
         DIGITALTWIN_CLIENT_ERROR_ACCESS_DENIED,                 \
         DIGITALTWIN_CLIENT_ERROR_CALLBACK_ALREADY_REGISTERED,   \
-        DIGITALTWIN_CLIENT_ERROR_DUPLICATE_INTERFACE_INSTANCES, \
+        DIGITALTWIN_CLIENT_ERROR_DUPLICATE_COMPONENTS, \
```

Comment:
>DIGITALTWIN_CLIENT_ERROR_DUPLICATE_COMPONENTS [](start = 8, length = 45)

General comment - please see digitaltwin_client\doc\interfaces.md line 12.  interfaceInstanceName also needs to be changed.  I didn't see anything else in the doc directory that needed cleaning up but please take a second look.